### ISSUE TITLE
Open channels when you're @mentioned in one

### ISSUE THREAD

**Issue by tekktonic** [sm00th/bitlbee-discord#210]
It'd be nice to autojoin channels when you're mentioned in them, since you can get so channel flooded in Discord.

**Comment by dgw** [comment]
Agreed! (But not for `@everyone` mentions <img src="https://i.imgur.com/VsuEEV9.gif" height="24px" alt=":angeryping:">)
